modules/apps/headless/headless-builder/headless-builder-test/src/testFunctional/tests/[UI]AllowUsersToDefineAPIApplicationEndpoints/EditAPIApplicationEndpoint.testcase

```diff
@@ -35,6 +35,67 @@ definition {
 		}
 	}
 
+	@priority = 3
+	test CanChangeSchemaOfEndpoint {
+		property portal.acceptance = "true";
+
+		task ("Given published API Application with Endpoint created") {
+			EndpointAPI.createAPIEndpoint(
+				apiApplicationERC = "My-app",
+				name = "testendpoint1",
+				path = "/testendpoint1");
+		}
+
+		task ("two API Schema in API Application") {
+			SchemaAPI.createAPISchema(
+				apiApplicationERC = "My-app",
+				mainObjectDefinitionERC = "L_API_APPLICATION",
+				name = "testSchema");
+		}
+
+		task ("When modifying endpoint again with API Schema schema2") {
+			APIBuilderUI.addAndPublishEndpointConfiguration(
+				entity = "Endpoints",
+				name = "testendpoint",
+				schemaName = "testSchema",
+				title = "My-app");
+
+			APIBuilderUI.addAndPublishEndpointConfiguration(
+				entity = "Endpoints",
+				name = "testendpoint1",
+				schemaName = "testSchema",
+				title = "My-app");
+		}
+
+		task ("Then changes are saved correctly") {
+			Alert.viewSuccessMessage();
```

Comment:
This is good to assert the success alert, and we also want to assert the schema for endpoint is actually updated.